package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Minimalistic browser-based Solidity IDE",
   "scripts": {
     "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
-    "test": "standard; babel src --out-dir babelify-src; node test/index.js",
+    "test": "babel src --out-dir babelify-src; node test/index.js && npm run lint",
```

Comment:
I think this should be `standard && babel src --out-dir babelify-src && node test/index.js` (or replace `standard` with `npm run lint`).